ls1intum/Artemis
# Show end confirmation page when exam is over

### Checklist

### Motivation and Context


Show exam end page before summary.

### Description

- Changed styles on cover page.
- Disabled confirmation box when waiting for exam start.
- Show exam end page on exam end (before summary is shown).
- fix keys and order of translations in exam.json for German

I did not check how the exam cover looks like when the instructor uses e.g. an image as start/end confirmation text or start/end text. I assumed that this will never happen

### Steps for Testing


1. Create an exam with start text start confirmation text, end text and end confirmation text
2. Create exercises, add students to exam, generate student exams and prepare exam
3. participate in exam - check exam start confirmation text, start text, end confirmation text and end text

### Known issue
- Nothing is displayed if an exam ends while the student is on the start exam confirmation page (exam-participation-cover) and the exam ends -> the student did not download the StudentExam yet -> nothing is displayed
- When reloading the exam after it has ended, the student has to accept the end terms again, because it is currently not saved to the server

### Screenshots



**Longer confirm text**
![image](https://user-images.githubusercontent.com/22955066/87029498-8cf62c80-c1e0-11ea-8ea2-76929e1c1fb9.png)

**Shorter confirm text**
![image](https://user-images.githubusercontent.com/22955066/87029658-b8791700-c1e0-11ea-95e1-ec3ec308c2ed.png)
